compiler/passes/convert-typed-uast.cpp

```diff
@@ -5071,6 +5090,157 @@ bool TConverter::enter(const ExternBlock* node, RV& rv) {
 
 void TConverter::exit(const ExternBlock* node, RV& rv) {}
 
+static SymExpr* makeCaseCond(TConverter& tc, TConverter::RV& rv,
+                             const uast::When* when,
+                             SymExpr* selectExpr,
+                             const uast::AstNode* cs) {
+  auto re = rv.byAst(cs);
+
+  // Grab the '==' ResolvedFunction
+  const AssociatedAction* action = nullptr;
+  for (const auto& a : re.associatedActions()) {
+    if (a.action() == AssociatedAction::COMPARE) {
+      action = &a;
+      break;
+    } else {
+      INT_ASSERT(false);
+    }
+  }
+  auto cmp = action->fn();
+
+  // TODO: create a wrapper for this kind of thing
+  const ResolvedFunction* rf = nullptr;
```

Comment:
I agree, maybe the wrapper can return the converted `FnSymbol*` or something as well as whether or not it was param elided, we might have enough uses of this to get a sense of the pattern here.

Reply:
I'd like to get a better idea of how this should work with the actual-converter, but some kind of wrapper seems inevitable.